cmd/gossamer/config.go

```diff
@@ -6,10 +6,12 @@ package main
 import (
 	"errors"
 	"fmt"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/ChainSafe/chaindb"
```

Comment:
I thought we were getting rid of this?

Reply:
This is just to remove the `utils.SetupDatabase` in favor of using the database implementation directly (since really that helper function isn't that necessary and adds complexity).